tests/integration/test_akafka.py

```diff
@@ -95,3 +98,75 @@ async def test_kafka_event_subscriber(kafka_fixture: KafkaFixture):  # noqa: F81
     translator.consume.assert_awaited_once_with(
         payload=payload, type_=type_, topic=topic
     )
+
+
+def find_kafka_secrets_dir() -> Path:
+    """Get the directory with Kafka secrets."""
+    current_dir = Path(__file__)
+    while current_dir != current_dir.parent:
+        current_dir = current_dir.parent
+        secrets_dir = current_dir / ".devcontainer" / "kafka_secrets"
+        if secrets_dir.is_dir():
+            for filename in "ca.crt", "client.crt", "client.key", "pwd.txt":
+                assert (secrets_dir / filename).exists(), (
+                    f"No {filename} in Kafka secrets directory."
+                    " Please re-run the create_secrets.sh script."
+                )
+                return secrets_dir
+    assert False, "Kafka secrets directory not found."
+
+
+@pytest.mark.asyncio
+async def test_kafka_ssl():
+    """Test connecting to Kafka via SSL.
+
+    This test uses the broker configured with the needed secrets via docker-compose
+    instead of a test container.
+    """
+    payload: JsonObject = {"test_content": "Be aware... Connect with care"}
+    type_ = "test_type"
+    key = "test_key"
+    topic = "test_topic"
+
+    admin_client = KafkaAdminClient(bootstrap_servers=["localhost:9092"])
```

Comment:
By convention, we are never using the docker-compose services of the .devcontainer for testing but always use testcontainers. Would you see an option to stick to that convention. Then we also would not have to modify the github actions.

Reply:
That's not so easy in this case, let's discuss offline.